packages/tx/src/constants.ts

```diff
@@ -3,7 +3,7 @@
 export const MAX_CALLDATA_SIZE = 16777216 // 2 ** 24
 export const MAX_ACCESS_LIST_SIZE = 16777216 // 2 ** 24
 export const MAX_VERSIONED_HASHES_LIST_SIZE = 16777216 // 2 ** 24
-export const LIMIT_BLOBS_PER_TX = 16777216 // 2 ** 24
+export const LIMIT_BLOBS_PER_TX = 6 // 786432 / 2^17 (`MAX_BLOB_GAS_PER_BLOCK` / `GAS_PER_BLOB`)
```

Comment:
i think we should move this constant to the teser/runner since this would be hf dependant anyway

Reply:
We can move `MAX_BLOB_GAS_PER_BLOB` and `GAS_PER_BLOB` into common?

Reply:
(and then derive max amount of blobs from that)